core/src/test/java/tech/tablesaw/api/FixDropDuplicatesTest.java

```diff
@@ -0,0 +1,38 @@
+package tech.tablesaw.api;
+
+import java.io.File;
+import java.lang.reflect.Method;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
+import it.unimi.dsi.fastutil.ints.Int2ObjectOpenHashMap;
+import it.unimi.dsi.fastutil.ints.IntArrayList;
+import tech.tablesaw.io.csv.CsvReadOptions;
+
+class FixDropDuplicatesTest {
```

Comment:
we've already got three tests for duplicates in TableTest.java. I think it'd be preferable to add any new test there alongside the existing tests